apps/python-sdk/firecrawl/v2/client_async.py

```
@@ -73,6 +77,57 @@ async def search(
+        Returns:
+            VectorSearchData containing the search results and metadata
+            
+        Raises:
+            ValueError: If request parameters are invalid
+            FirecrawlError: If the search operation fails
+        """
+        # Convert filters dict to VectorSearchFilters if provided
+        filters_obj = None
+        if filters is not None:
+            filters_obj = VectorSearchFilters(**filters)
+            
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)